src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -324,9 +323,6 @@ public void simulationPeriodic() {
         // update the DutyCycleEncoder
         encoderSim.setDistance(simulation.getAngleRads());
 
-        // Calculate the current drawn by one of the motors
-        double ampsPerMotor = simulation.getCurrentDrawAmps() / 4;
-
         // see https://docs.google.com/spreadsheets/d/1UiHZFYeZiHPAPIu39uRrskQuQYfvJ03UjLeQVq--Mzg/edit#gid=0
         // Arm motors uses channels 1, 2, 4, 5
         // m_powerPanel.setCurrent(1, ampsPerMotor);
```

Comment:
Do not delete these (former 327-329).
These want to be used with the PowerPanel to simulate current drawn from the battery -- a topic that needs to be understood.

Reply:
The team needs to change its plans. For two years in a row, the team has drawn too much current.

Reply:
Yes, and I agree that tracking that current is important. However, to accurately simulate the current and voltage of the entire robot, we would need to rewrite the drivetrain sim, which takes more current throughout a match than the arm. While we should rewrite that anyway, we need to focus on advanced projects first. Like I said in my last comment, we can reimplement ;something similar to PowerPanel during build season after we know what subsystems we will use instead of spending time now adding it to our 2024 subsystems.

Reply:
I disagree.

1. PowerPanel works on the real robot (modulo a grievious programming error). The current report is useful even without simulation.
2. Learning how to simulate current now is important. If you shove it under the carpet, then you will not have time during build season.
3. Why shouldn't the elevator sim and turret sim calculate the current draw during simulation? We are talking a few lines of code. Isn't that one of the purposes of having an advanced project?

Reply:
<ol>
<li>I think there are some existing method to get the robot's voltage and current, such as the ones in RobotController.</li>
<li>I agree that it is important, but we don't have time before the advanced project deadline to work on something that we need to add, test, and debug for all subsystems. An accurate swerve simulation that uses current would take time that we don't have right now (we could maybe do this after advanced project finish). Also, for the other subsystems, like you said, "We are talking a few lines of code", so we can easily add it as we create our subsystems next year.</li>
<li>Turret and elevator could use it, although they don't take a lot of current. Assuming hardware finished both of them relatively soon, there should be time to add this. However, this code is for the arm.</li>
</ol>